components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/validator/DefaultApplicationValidator.java

```diff
@@ -63,9 +63,9 @@
 /**
  * Validator class to be used to validate the consistency of the Application/Service Provider, before it is persisted.
  */
-public class ApplicationMgtValidator {
+public class DefaultApplicationValidator implements ApplicationValidator {
```

Comment:
Should we do a version bump for this change?

Reply:
sure.